uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1393,21 +1768,20 @@ sub getUploadIDUsingTarchiveSrcLoc {
     return $upload_id;
 }
 
-################################################################
-#################spool##########################################
-################################################################
+
 =pod
-spool()
-Description:
-   - Calls the Notify->spool function to log all messages
-
-Arguments:
- $this      : Reference to the class
- $message   : Message to be logged in the database
- $error     : if 'Y' it's an error log , 'N' otherwise
- $upload_id: The upload_id
- $verb      : 'N' for few main messages, 'Y' for more messages (developers)
- Returns    : NULL
+
+=head3 spool($message, $error, $upload_id, $verb)
+
+Calls the Notify->spool function to log all messages.
+
+INPUT:
+  $message   : message to be logged in the database
+  $error     : if 'Y' it's an error log , 'N' otherwise
```

Comment:
just please put the 'N' statement under the 'Y'. We did that for other scripts (and you have it like this for the `$verb` option)